### DISCUSSION TITLE
Visualize Scaled Beam Deformations

### DISCUSSION THREAD

**Discussion by SimonAndreas** [jgerstmayr/EXUDYN#70] [Q&A on usage of Exudyn]
Currently I am using 2D beam elements (ObjectBeamGeometricallyExact2D) and I would like to visualize scaled beam deformations (e.g. exaggerated beam deflection) such that it is easier to interpret small deformations using the SolutionsViewer(). 

Should I use “SC.visualizationSettings.bodies.deformationScaleFactor” for this purpose or does a different function exist? Namely, I tried the beforementioned function but unfortunately nothing changed in the visualization; the deformations were still very small. Is it possible that I made a mistake when using this setting? Thank you in advance.

**Comment by jgerstmayr** [comment]
moved to issue #71
